js/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "include": ["src/**/*.ts"],
+  "include": ["src/**/*"],
```

Comment:
This feels quite broad, essentially saying any file in src. Couldn't this cause files unrelated to JS/TS/TSX to be recognized incorrectly?

Reply:
@askvortsov1, according to the official documentation, this will actually do what we want it to do - parse only TypeScript files:

> If a glob pattern doesn’t include a file extension, then only files with supported extensions are included (e.g. .ts, .tsx, and .d.ts by default, with .js and .jsx if allowJs is set to true).
https://www.typescriptlang.org/tsconfig#include

Since allowJs is not explicitly set to true, this change will mean that TS will go from parsing just `.ts` files to `.ts`, `.tsx`, and `d.ts`, ignoring JS files and anything else that may be present in those directories.